Control coverage/Feature/SubscriptionCore.md

```diff
@@ -1,14 +1,933 @@
+
+### Control Spec 
+
+> **Passed:** 
+> No external account is found at subscription scope.
+> 
+> **Failed:** 
+> External account is found at subscription scope.
+> 
+> **Verify:** 
+> RBAC result not found (sufficient data is not available for evaluation).
+> 
+
+### Recommendation 
+
+- **PowerShell** 
+
+	 ```powershell 
+	 Remove-AzRoleAssignment -SignInName '{signInName}' -Scope '{scope}' -RoleDefinitionName '{role definition name}'
+     # Run 'Get-Help Remove-AzRoleAssignment -full' for more help.
+     # For bulk remediation using PowerShell, refer https://aka.ms/azts-docs/rscript/Azure_Subscription_AuthZ_Dont_Use_NonAD_Identities
+	 ```  
+
+### Azure Policy or ARM API used for evaluation 
+
+- PIM API to get role assignments: - /beta/privilegedAccess/azureResources/resources/{uniquePIMIdentifier}/roleAssignments?$expand=subject,roleDefinition($expand=resource)&$filter=(memberType ne '{filterCondition}')<br />
+**Properties:** subject.principalName
+ <br />
+
+- ARM API to list classic role assignment at subscription level: - /subscriptions/{subscriptionId}/providers/Microsoft.Authorization/classicAdministrators?api-version=2015-06-01<br />
+**Properties:** emailAddress
+ <br />
+
+<br />
+
+___ 
+
+## Azure_Subscription_AuthZ_Dont_Use_NonAD_Identities_Privileged_Roles 
+
+### DisplayName 
+Remove external accounts with privileged roles at subscription scope 
+
+### Rationale 
+Non-AD accounts (such as xyz@hotmail.com, pqr@outlook.com, etc.) present at any scope within a subscription subject your cloud assets to undue risk. These accounts are not managed to the same standards as enterprise tenant identities. They don't have multi-factor authentication enabled. 
+
+### Control Settings 
+```json 
+{
+    "PrivilegedRoles": [
+        "User Access Administrator",
+        "Owner",
+        "Contributor"
+    ]
+}
+ ```  
+
+### Control Spec 
+
+> **Passed:** 
+> No external account with privileged roles at Subscription scope found.
+> 
+> **Failed:** 
+> One or more external account(s) found with privileged roles at Subscription scope.
+> 
+> **Verify:** 
+> RBAC details for the Subscription are not available.
+> 
+
+### Recommendation 
+
+- **PowerShell** 
+
+	 ```powershell 
+	 Remove-AzRoleAssignment -SignInName '{signInName}' -Scope '{scope}' -RoleDefinitionName '{roleDefinitionName}'
+     # Run 'Get-Help Remove-AzRoleAssignment -full' for more help. 
+	 ```  
+
+### Azure Policy or ARM API used for evaluation 
+
+- PIM API to get role assignments: - /beta/privilegedAccess/azureResources/resources/{uniquePIMIdentifier}/roleAssignments?$expand=subject,roleDefinition($expand=resource)&$filter=(memberType ne '{filterCondition}')<br />
+**Properties:** subject.type, roleDefinition.displayName
```

Comment:
Should this be `[*].properties...`, considering the response is a list of assignments. `[*]` to indicate - for all assignments.